### ISSUE TITLE
中途为啥test accurary为0.0啊？

### ISSUE THREAD

**Issue by Vergissmeinnic** [sirius-ai/LPRNet_Pytorch#74]
训练一开始还是挺正常的，就是中途test accurary为0.0了，而且最后的best模型也很低，只有0.44

**Comment by Vergissmeinnic** [comment]
缩小了learning_rate之后到0.75左右了，但是始终无法突破0.8

**Comment by kankanjiuzou123** [comment]
我的一直都是0.0

**Comment by feitianhouren** [comment]
Set the training epochs parameter to a larger value (for example, 100), increase the test frequency (decrease the --test_interval parameter), and carefully observe the accuracy of each test to see if there are non-zero values. In my training, using my own data set (80,000 for training set and 20,000 for test set), after 15 iterations of the network, the accuracy has reached more than 0.94 (the learning rate is 0.00001 at this time), and then continue training (set The epochs value is greater than 15), the train_*.py program will automatically adjust the learning rate back to 0.1, the loss will become larger again, and the accuracy will be reduced to 0.

**Comment by yz80325** [comment]
> 



> 我的一直都是0.0

最后你怎么解决的呢？我也是

**Comment by yutao007** [comment]
减少训练测试，看到作者在lr的变化区间就到16epoch.所以最好就跑15个epoch。否则调整变化区间
